### PR TITLE
Barium data

### PR DESCRIPTION
**Pull request by sotirova**
Add atomic structure data for $\mathrm{^{138}Ba^+}$, $\mathrm{^{137}Ba^+}$, $\mathrm{^{135}Ba^+}$, and $\mathrm{^{133}Ba^+}$.

### CODE REVIEW

File: atomic_physics/common.py [hunk 1]
```diff
@@ -406,7 +406,7 @@ def setB(self, B: float):
 
             for M in set(self.M[lev]):
                 for Fidx, idx in np.ndenumerate(np.where(M == self.M[lev])):
-                    self.F[lev][idx] = F_list[M <= F_list][Fidx[1]]
+                    self.F[lev][idx] = F_list[abs(M) <= F_list][Fidx[1]]
```
Comment (hartytp) [inline atomic_physics/common.py:409]: Wasn't that fixed in a separate PR?

File: atomic_physics/ions/ba133.py [hunk 2]
```diff
@@ -0,0 +1,132 @@
+""" 133Ba+
+
+ToDo: We need a reference for the Einstein A values
```
Comment (hartytp) [inline atomic_physics/ions/ba133.py:3]: can you open an issue for this and remove the TODO so it doesn't get forgotten about plz?

File: atomic_physics/ions/ba133.py [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+A cursory look turned up:
+    Kramida, A., et al., NIST Atomic Spectra Database (ver. 5.8), [Online].
+However, this data is based of calculations done in the late 60s. These are
+only ~10% accurate.
+
+The transition frequencies are calculated based on the 138Ba+ frequencies
+from [1] and the isotope shifts in the second reference listed next to
+the freqiency. Where no references are given, the transition frequencies
```
Comment (hartytp) [inline atomic_physics/ions/ba133.py:11]: spelling

**Review by hartytp — APPROVED** [review]
Thanks! A couple of nit picks but overall looks good.

So many barium isotopes!

File: atomic_physics/common.py (hunk 1 above)
Comment (sotirova) [reply atomic_physics/common.py:409]: Yes - though this PR came from the same branch that had all the Ba isotopes as well so the change is here too (forgot to remove...).

File: atomic_physics/ions/ba133.py (hunk 3 above)
Comment (sotirova) [reply atomic_physics/ions/ba133.py]: Fixed

**Pull request merged**